cardano-cli/src/Cardano/CLI/EraBased/Query/Option.hs

```diff
@@ -359,13 +362,21 @@ pQueryTipCmd era envCli =
       <$> pQueryCommons era envCli
       <*> pMaybeOutputFile
 
+fFrom :: Functor f => a :| l => f a -> f (Vary l)
+fFrom = fmap from
+
 pQueryUTxOCmd :: ShelleyBasedEra era -> EnvCli -> Parser (QueryCmds era)
 pQueryUTxOCmd era envCli =
   fmap QueryUTxOCmd $
     QueryUTxOCmdArgs
       <$> pQueryCommons era envCli
       <*> pQueryUTxOFilter
-      <*> (optional $ pAllOutputFormats "utxo")
+      <*> ( optional $
+              empty
```

Comment:
Why not:
```
              asum
                [ fFrom (pFormatCBOR "utxo")
                , fFrom (pFormatJsonFileDefault "utxo")
                , fFrom (pFormatTextStdoutDefault "utxo")
                ]
```

Reply:
`asum` is good. I'll change it in the next PR.